src/Select/index.js

```diff
@@ -16,6 +16,10 @@ const sharedStyle = css`
     border-color: ${t('colors.grays.2')};
   }
 
+  .ut-select__menu {
+    z-index: 10000000;
```

Comment:
This is one zero less than the modal z-index, in case you were wondering

Reply:
Ha! I wasn't exactly advocating for putting these on the moon 🤣 The select menu being open and on top of an alert/modal seems like something that won't happen? Either way, this is fine if you think it's fine. My original comment was more or less about going from 4 -> 1000 instead of 4 -> 40 to give a little more gap. Fine with whatever here 👍

Reply:
I looked at the most common z-indexes in our codebase, and most of them start at 10,000 :no_mouth. So I just wanted to make sure these would work with what we already have.